src/sections/Community/Member-single/index.js

```diff
@@ -274,6 +274,13 @@ const MemberSingle = ({ frontmatter }) => {
                       </Link>
                     </li>
                   )}
+                  {
+                    <li>
+                      <a href="https://layer5.io/cloud-native-management/meshery" target="_blank" rel="noreferrer">
+                        <img src={mesheryLogo} alt="meshery-icon"></img>
```

Comment:
Nice work @Manish254252. Can we add the white version of the logo here to make things consistent? The rest looks good to me. Thank you @Manish254252.

Reply:
this is adding meshery-cloud to every member profile , we wish to add only on users who are present in meshery cloud

Reply:
right @Ghat0tkach